js/tests/enhancedDate.spec.js

```diff
@@ -0,0 +1,120 @@
+
+//Brian Forcum, Module_6 Homework
+
+describe("enhancedDate", function () {
+    
+    'use strict';
+
+    describe("setDate", function () {
+    
+        beforeEach(function () {
+                enhancedDate.setDate(new Date(2015, 4, 14));
+            });
+
+        it("Should be an instance of Date", function () {
+           expect(enhancedDate.setDate()).is.an.instanceof(Date);
+        });
+
+        it("Should not return null", function () {
+            expect(enhancedDate.setDate()).not.to.equal(null);
```

Comment:
This test is redundant because you already know the return value is an instance of Date. A second test could instead, for example, check that the return value of setDate was deep.equal to a new Date object instantiated with the same values you called setDate with originally.